include/deal.II/multigrid/multigrid.h

```diff
@@ -74,6 +74,60 @@ class Multigrid : public Subscriptor
     f_cycle
   };
 
+  /**
+   * A structure that has boost::signal objects for a number of actions that
+   * happen for the typical usage of this class.
+   *
+   * For documentation on signals, see
+   * http://www.boost.org/doc/libs/release/libs/signals2 .
+   */
+  struct Signals
+  {
+    /**
+     * This signal is triggered before and after the call to
+     * MGTransfer::copy_to_mg which transfers given vector to all levels.
```

Comment:
which vector does this refer to? The vector given to `copy_to_mg`? If so, maybe make that clearer.